### DISCUSSION TITLE
Should be able to darken text and icons when they're on top of your reddit background

### DISCUSSION THREAD

**Discussion by AlmightyWings232** [Docile-Alligator/Infinity-For-Reddit#1076] [Ideas]
There should be an option in the settings (maybe in interface or themes) that lets you darken the text and icons that are on top of your reddit background, because it's hard to see them if you have a bright background. I've attached some images showing the text and icons. 

## Screenshots

![Screenshot_20210520_173607](https://user-images.githubusercontent.com/44907494/119017986-7dfd1080-b993-11eb-9687-57f25fd486d5.jpg)
![Screenshot_20210520_174050](https://user-images.githubusercontent.com/44907494/119017990-805f6a80-b993-11eb-8345-53d140f213f6.jpg)

**Comment by Docile-Alligator** [comment]
Hi, the screenshots are not shown. Do you mean to make the icons and texts on the toolbar have black color in subreddit page?

**Comment by AlmightyWings232** [comment]
Hmmm, that's odd. The screenshots show fine for me, it should be a screenshot with an orange circle showing the text/icons which I think should have the option to be darkened (since they're hard to read, at least with a bright background). Maybe try clicking these links instead then, they should hopefully load the screenshots:

https://user-images.githubusercontent.com/44907494/119017986-7dfd1080-b993-11eb-9687-57f25fd486d5.jpg

https://user-images.githubusercontent.com/44907494/119017990-805f6a80-b993-11eb-8345-53d140f213f6.jpg
